remoteappmanager/docker/container.py

```diff
@@ -32,12 +32,22 @@ class Container(HasTraits):
     #: the id that will go in the URL of the container
     url_id = Unicode()
 
+    #: The absolute base prefix path for the container.
+    base_urlpath = Unicode(None, allow_none=True)
```

Comment:
because when it's created, it is None. The creation happens at the container manager level, but the container manager has no clue of where the application is based on. This is really something I don't like, but I could not find a decent solution for it. I don't want the container manager to know the application, nor the application base. One alternative would be to tell the reverse proxy to add the base url, but then you need the absolute url regardless, when you have to do the redirection.